[Workers] Document wrangler's proxy support (#5047)
cloudflare/cloudflare-docs: content/workers/wrangler/configuration.md

  { binding = "TEST_BINDING", service = "", environment = "" }
]
```

## Proxy Support

> Wrangler supports the use of proxies via environment variables. 

Reviewer: Need to answer "why" would a dev do this and "how".

> Wrangler supports the use of proxies via environment variables. 

Author: ```suggestion
When using Wrangler in corporate environments that require the use of a VPN or HTTP proxy, you may come across a `UNABLE_TO_GET_ISSUER_CERT_LOCALLY` error.

You can use Wrangler with your VPN or HTTP proxy via environment variables. 
```

> The first non-empty environment variable named https_proxy, HTTPS_PROXY, http_proxy or HTTP_PROXY will be used when wrangler makes outgoing requests.

Author: ```suggestion
The first non-empty environment variable named `https_proxy`, `HTTPS_PROXY`, `http_proxy` or `HTTP_PROXY` will be used when wrangler makes outgoing requests.
```

> The first non-empty environment variable named `https_proxy`, `HTTPS_PROXY`, `http_proxy` or `HTTP_PROXY` will be used when wrangler makes outgoing requests.

Reviewer: Is this an action that the user is doing or observing? It needs directions if this is something the user is doing. For example:

To __(goal:what is dev doing?)___, go to __(place: dashboard? terminal?)___, and ___(action that dev needs to do)___.

> The first non-empty environment variable named `https_proxy`, `HTTPS_PROXY`, `http_proxy` or `HTTP_PROXY` will be used when wrangler makes outgoing requests.

Author: ```suggestion
To use Wrangler behind a proxy, you'll need to set an environment variable pointing to your proxy.

Wrangler supports the following environment variable names:
- `https_proxy`
- `HTTPS_PROXY`
- `http_proxy`
- `HTTP_PROXY`

The first non-empty environment variable in this list will be used when wrangler makes outgoing requests.
```

> For example, on macOS this can be done by adding `HTTP_PROXY=http://<YOUR_PROXY_HOST>:<YOUR_PROXY_PORT>` before your wrangler commands, as shown below:

Reviewer: Do you need to set up environment variables before doing this step?

  { binding = "TEST_BINDING", service = "", environment = "" }
]
```

## Proxy support

When using Wrangler in corporate environments that require the use of a VPN or HTTP proxy, you may come across a `UNABLE_TO_GET_ISSUER_CERT_LOCALLY` error, or your IT team may ask you to use a proxy.

You can use Wrangler with your VPN or HTTP proxy via environment variables. 

Wrangler supports the following environment variable names:
- `https_proxy`
- `HTTPS_PROXY`
- `http_proxy`
- `HTTP_PROXY`

> For example, to configure this on macOS add `HTTP_PROXY=http://<YOUR_PROXY_HOST>:<YOUR_PROXY_PORT>` before your wrangler commands, as shown below:

Author: ```suggestion

For example, to configure this on macOS add `HTTP_PROXY=http://<YOUR_PROXY_HOST>:<YOUR_PROXY_PORT>` before your wrangler commands, as shown below:
```

> The first non-empty environment variable in this list will be used when Wrangler makes outgoing requests.

Author: ```suggestion
```

  { binding = "TEST_BINDING", service = "", environment = "" }
]
```

## Proxy support

> When using Wrangler in corporate environments that require the use of a VPN or HTTP proxy, you may come across a `UNABLE_TO_GET_ISSUER_CERT_LOCALLY` error.

Author: ```suggestion
When using Wrangler in corporate environments that require the use of a VPN or HTTP proxy, you may come across a `UNABLE_TO_GET_ISSUER_CERT_LOCALLY` error, or your IT team may ask you to use a proxy.
```

You can use Wrangler with your VPN or HTTP proxy via environment variables. 

Wrangler supports the following environment variable names:
- `https_proxy`
- `HTTPS_PROXY`
- `http_proxy`
- `HTTP_PROXY`

> 

Author: ```suggestion
For example, to configure this on macOS add `HTTP_PROXY=http://<YOUR_PROXY_HOST>:<YOUR_PROXY_PORT>` before your wrangler commands, as shown below:
```

> The first non-empty environment variable in this list will be used when wrangler makes outgoing requests.

Reviewer: ```suggestion
The first non-empty environment variable in this list will be used when Wrangler makes outgoing requests.
```

> For example, on macOS this can be done by adding `HTTP_PROXY=http://<YOUR_PROXY_HOST>:<YOUR_PROXY_PORT>` before your wrangler commands, as shown below:

Author: ```suggestion
```

  { binding = "TEST_BINDING", service = "", environment = "" }
]
```

## Proxy support

> When using Wrangler in corporate environments that require the use of a VPN or HTTP proxy, you may come across a `UNABLE_TO_GET_ISSUER_CERT_LOCALLY` error, or your IT team may ask you to use a proxy.
> 

Reviewer: ```suggestion
```

You can use Wrangler with your VPN or HTTP proxy via environment variables. 

> 

Author: ```suggestion
```

> You can use Wrangler with your VPN or HTTP proxy via your computer's environment variables. 

Author: ```suggestion
You can use Wrangler with your VPN or HTTP proxy by configuring your computer's environment variables. 
```

> You can use Wrangler with your VPN or HTTP proxy by configuring your computer's environment variables. 

Reviewer: ```suggestion
To enable proxy setup, use Wrangler with your VPN or HTTP proxy by configuring your computer's environment variables. Note that when using Wrangler in corporate environments that require the use of a VPN or HTTP proxy, you may come across a `UNABLE_TO_GET_ISSUER_CERT_LOCALLY` error. This error can be fixed with the use of environment variables.
```

Reviewer: Because proxy support is the main purpose of this paragraph, I rearranged this section to align with that.

> To use Wrangler behind a proxy, you'll need to set an environment variable pointing to your proxy.

Author: ```suggestion
```

> To use Wrangler behind a proxy, you'll need to set an environment variable pointing to your proxy.

Author: ```suggestion
```
